include/deal.II/grid/tria.h

```diff
@@ -1893,6 +1893,11 @@ class Triangulation : public Subscriptor
    * assignment of a different Manifold object by the function
    * Triangulation::set_manifold().
    *
+   * @note Geometric objects having the manifold ID @p manifold_number will
+   *   after calling this function still have the same ID. Their ID are not
+   *   replaced by numbers::flat_manifold_id so that the function
+   *   get_manifold_ids() will still return the same set of IDs.
```

Comment:
```suggestion
   * @note Geometric objects with the manifold ID @p manifold_number will
   *   still have the same ID after calling this function so that the function
   *   get_manifold_ids() will still return the same set of IDs.
```

Reply:
Thanks for the suggestion. I have applied the suggestion.